wasm-engine/tests/web.rs

```diff
@@ -1,12 +1,12 @@
 //! Test suite for the Web and headless browsers.
 
-#![cfg(target_arch = "wasm32")]
+#![cfg(target_family = "wasm")]
```

Comment:
`target_family = "wasm"` should have more coverage than specifically `target_arch = "wasm32"`, right?

Reply:
Sure! 64 bit WASM is still in experimental but I don't see harm in opening that up a bit